### ISSUE TITLE
update browserify to version 16

### ISSUE THREAD

**Issue by khalyomede** [ngryman/gulp-bro#60]
Could you update your package.json (if it is enough?) browserify version to 16, in order to be able to use split-require with your gulp plugin please?

**Comment by ngryman** [comment]
Hey @khalyomede ,

I've upgraded all dependencies (including `browserify`).
Would mind giving it a try before I officially publish it?

To test, you can modify your `package.json` like this:

```json
{
  "devDependencies": {
    "gulp-bro": "ngryman/gulp-bro#bump/all-deps"
  }
}
```

Let me know if it works.

**Comment by khalyomede** [comment]
Hi @ngryman, it works at the perfection :) I checked by seeing if [split-require](https://github.com/goto-bus-stop/split-require) worked (and as my SPA is heavily using dynamic import, I also checked if it still works to navigate between pages).

So for now all good! Thank you 👍

**Comment by ngryman** [comment]
@khalyomede Alright, happy it works as expected. I've just released `2.0.0` so you can change your dependency back to `semver` if you want.
